themes/SuiteP/css/suitep-base/suitepicon-glyphs.scss

```diff
@@ -361,6 +361,11 @@
 .suitepicon-mimetype-tab:before {
     content: "\f179";
 }
+// do not remove this
+[class*="suitepicon-module"]:before,
+.suitepicon-module-default:before {
```

Comment:
Please place fixes in the suitepicon.scss file. As the suitepicon-glyphs are auto generated and the value of the content property will change each time you regenerate the fonts. 

Please use the following code in the suitepicon.scss file.
```css
[class*="suitepicon-module"]:before {
    @extend .suitepicon-module-default;
}
```

This code will always contain the correct content value each time the font is regenerated.

Reply:
Fixed.